src/coreclr/System.Private.CoreLib/src/System/Runtime/JitInfo.CoreCLR.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using Internal.Runtime.CompilerServices;
+
+namespace System.Runtime
+{
+    public static partial class JitInfo
```

Comment:
Nit: the type itself also needs an XML comment.

Reply:
Added a short blurb.